package-rust.py

```diff
@@ -176,15 +180,26 @@ def run(args):
 if channel == "stable":
     CFG_RELEASE=CFG_RELEASE_NUM
     CFG_PACKAGE_VERS=CFG_RELEASE_NUM
+    # UpgradeCode shoud stay the same for all MSI versions in channel
+    CFG_UPGRADE_CODE="1C7CADA5-D117-43F8-A356-DF15F9FBEFF6"
```

Comment:
Do these upgrade codes mean that the MSI version of rust release channels can be installed simultaneously? Do the installers modify PATH in conflicting ways?